### PR TITLE
INS-1576: cleanup and simplify launchnet

### PR DESCRIPTION
**Pull request by nordicdyno**
* write dynamic files to .artifacts/launchnet directory by default
* remove a lot of outdated logic, try to simplify that's left

### CODE REVIEW

File: functest/main_test.go [hunk 1]
```diff
@@ -96,12 +97,24 @@ func functestPath() string {
...
+	value := os.Getenv(name)
+	if value != "" {
+		return value
+	}
+	return defaultValue
 }
 
-func deleteDirForContracts() error {
-	return os.RemoveAll(filepath.Join(functestPath(), "contractstorage"))
+// func baseDir() string {
```
Comment (ivanshibitov) [inline functest/main_test.go]: remove it

File: bootstrap/genesis/init.go [hunk 2]
```diff
@@ -133,14 +133,14 @@ func (s *Initializer) Run() {
...
 	inslog, err := log.NewLog(cfg)
+
 	if err != nil {
 		panic(err)
 	}
 
-	if newInslog, err := inslog.WithLevel(cfg.Level); err != nil {
+	inslog, err = inslog.WithLevel(cfg.Level)
+	if err != nil {
 		inslog.Error(err.Error())
```
Comment (megge-dream) [inline bootstrap/genesis/init.go]: `inslog` can became `nil` here

File: functest/main_test.go (hunk 1 above)
Comment (megge-dream) [inline functest/main_test.go]: Can we remove this?

File: scripts/insolard/check_status.sh [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
-confs=$( dirname $0 )"/configs/generated_configs/discoverynodes"
-api_ports=$( grep -A 1 "apirunner" $confs/insolar_*.yaml  | grep address  |  grep -o ":\d\+" | grep -o "\d\+" | tr '\n' ' ' )
+# Changeable environment variables (parameters)
+INSOLAR_ARTIFACTS_DIR=${INSOLAR_ARTIFACTS_DIR:-".artifacts"}/
+LAUNCHNET_BASE_DIR=${LAUNCHNET_BASE_DIR:-"${INSOLAR_ARTIFACTS_DIR}launchnet"}/
+
+# check is discovery nodes ready
+
+confs=${LAUNCHNET_BASE_DIR}discoverynodes/
+#(>&2 echo "grep -A 1 'apirunner' ${confs}*/insolard.yaml  | grep address  |  grep -o ':\d\+' | grep -o '\d\+' | tr '\n' ' '")
```
Comment (ivanshibitov) [inline scripts/insolard/check_status.sh]: do we need it?

**Pull request merged**